Adjust test_eacl_can_not_be_set_with_final_bit to current node

https://rest.fs.neo.org/Aku38wpn2xEqfEPxmwyAa7N3VnAkMV8PEVkpLqwKM1z3/5141-1775829634/index.html#suites/a52f046dd0c985c0a228b52b76fd305f/1b9f7cb23cfd09b1/

> AssertionError: Regex pattern did not match.
 Regex: '.*timeout.*'
 Input: "Command: ./neofs-cli --config /Users/runner/work/neofs-node/neofs-node/neofs-testcases/wallet_config.yml container set-eacl --rpc-endpoint 'localhost:49277' --wallet '/Users/runner/work/neofs-node/neofs-node/neofs-testcases/test-run-2026-04-10-11-28-00-230242/wallet-c082947f-a352-4148-aafa-de42b31105a2.json' --cid '2qoo8aNFMt8DmbeurPozJJr3jDvB2y48z3hTpcoDBTnk' --await --table '/Users/runner/work/neofs-node/neofs-node/neofs-testcases/test-run-2026-04-10-11-28-00-230242/TestFilesDir/eacl_table_a2b9dffe-3c2d-470d-9522-68fcf5323573.json' --force\nError:\nreturn code: 1\noutput: Flag --await has been deprecated, operation is synchronous now. Use 'neofs-cli container get-eacl' to double-check success instead.\nError: rpc error: container setEacl: status: code = 1024 message = fallback transaction accepted\n"

Changed since https://github.com/nspcc-dev/neofs-node/pull/3868, both errors are fine.

--- a/pytest_tests/lib/helpers/grpc_responses.py
+++ b/pytest_tests/lib/helpers/grpc_responses.py
@@ -49,6 +49,7 @@ EACL_NOT_FOUND = "code = 3073.*message = eACL not found"
 EACL_PROHIBITED_TO_MODIFY_SYSTEM_ACCESS = "table validation: it is prohibited to modify system access"
 EACL_CHANGE_PROHIBITED = ".*container ACL is immutable.*"
 EACL_CHANGE_TIMEOUT = ".*timeout.*"
+EACL_CHANGE_RPC_ERROR = ".*rpc error.*"
 
 INVALID_SEARCH_QUERY = ".*invalid search query.*"
 INVALID_RULES = ".*unable to parse provided rules.*"

--- a/pytest_tests/tests/acl/test_eacl.py
+++ b/pytest_tests/tests/acl/test_eacl.py
@@ -19,6 +19,7 @@ from helpers.container import EC_3_1_PLACEMENT_RULE, create_container
 from helpers.container_access import check_full_access_to_container, check_no_access_to_container
 from helpers.grpc_responses import (
     EACL_CHANGE_PROHIBITED,
+    EACL_CHANGE_RPC_ERROR,
     EACL_CHANGE_TIMEOUT,
     EACL_PROHIBITED_TO_MODIFY_SYSTEM_ACCESS,
     NOT_CONTAINER_OWNER,
@@ -106,7 +107,7 @@ class TestEACLContainer(TestNeofsBase):
         with allure.step("Try to set eacl for such container with force"):
             eacl_deny = [EACLRule(access=EACLAccess.DENY, role=EACLRole.USER, operation=op) for op in EACLOperation]
 
-            with pytest.raises(RuntimeError, match=EACL_CHANGE_TIMEOUT):
+            with pytest.raises(RuntimeError, match=rf"{EACL_CHANGE_TIMEOUT}|{EACL_CHANGE_RPC_ERROR}"):
                 set_eacl(
                     user_wallet.wallet_path,
                     cid,

